docs/_docs/Chrome/cookie.md

```diff
@@ -0,0 +1,64 @@
+---
+title: .cookie
+category: Chrome
+---
+
+The `cookie` method acts as a getter/setter for accessing and setting cookies.
+
+When executed with no arguments, the `cookie` method will return an objects representing the browser cookies. They will contain a `name` property (the name of the cookie) and it's respective value.
```

Comment:
Does this mean to say `... will return an array of objects representing...` here?